<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>371</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Select_T1">
      <property name="text">
       <string>Select Team 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Select_T2">
      <property name="text">
       <string>Select Team 2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="Stats_team1_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Team_1">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QTableWidget" name="Stats_team1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>151</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>TEAM ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SECONDS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FGM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FGA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FG3M</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FG3A</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FTM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FTA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>OREB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>AST</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STL</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BLK</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TURN OVER</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PF</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PTS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>REB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>GAME ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PLAYER ID</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="Team_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QTableWidget" name="Stats_team2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>151</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>TEAM ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SECONDS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FGM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FGA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FG3M</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FG3A</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FTM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FTA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>OREB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>AST</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STL</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BLK</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TURN OVER</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PF</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PTS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>REB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>GAME ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PLAYER ID</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
